docs/applications/deploy-application-using-incoming-hook.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Deploy Application using an Incoming Hook
```

Comment:
The same way we separated SSL, DNS and `find server IP address`, I believe here we should first document an independent topic guiding users on how to create incoming webhooks (`/docs/webhooks/create-incoming-webhook.md`) and then here go through the steps that are specific to application deployments, with link to the common `create-incoming-webhook` steps.

Suggestion of intro text for `create-incoming-webhook`:
```
title: Create Incoming Webhooks
intro: Incoming Webhooks are a simple way to integrate external services with your Devopness environments, 
using a unique URL through which a JSON payload can be sent to run a pipeline with custom input data. 
Some use cases of external services events include "deploy applications automatically based on 
GitHub/GitLab/Bitbucket repository updates", "provision and scale infrastructure resources when you 
payment gateway confirms that a custom subscription has been processed", and many more use cases 
that can benefit from webhooks integration. Learn how to reate an incoming webhook to run pipelines 
programmatically when events happen on external services.
```

Reply:
@jfoliveira PR updated